InventoryComponents/InventoryConsumablesComponent.gd

```diff
@@ -1,7 +1,9 @@
 class_name InventoryConsumablesComponent
 
-##The held_consumables array has slots with the value null so that we can [br]
-## have item in for exsample slot 1 and 3 but nothing in slot 2
+## Inventory Component resposible for Consumables [br]
+## There is a set amount of slots the player has and each of them can have one item, if any items are given to the player when all slots are full then they dissapear
```

Comment:
if any item is given to the player when all slots are full, then the item disappears